app/src/main/res/values/strings.xml

```diff
@@ -254,7 +254,8 @@
   <string name="copy_wikicode">Copy the wikitext to the clipboard</string>
   <string name="wikicode_copied">The wikitext was copied to the clipboard</string>
   <string name="nearby_location_not_available">Nearby might not work properly, Location not available.</string>
-  <string name="location_permission_rationale_nearby">Permission required to display a list of nearby places</string>
+  <string name="location_permission_rationale_nearby">The Commons app collects location data to
+    display nearby places that need photos even when the app is in background.</string>
```

Comment:
I would say "in the background".
I unfortunately can't test right now, but apart from this it looks fine so feel free to merge. 🙂

Reply:
and maybe "photos," instead of "photos"? Comma removes an ambiguity and makes the sentence easier to read, I think.